notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    !pip install RLHive['gym_minigrid']
</pre><p>This should be a separate PR, but it should probably be merged before this one. gym_minigrid is now just minigrid, and it's available as part of the Farama Foundation org. If you could update this tutorial to reflect those changes, that would be great. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>So should I wait until the new PR is merged? Or I should install by directly installing Minigrid by running <code>pip install minigrid</code>?</p>